syntax = "proto3";

message DSItem {
    string ProcessedQuery = 1;
    string GeoData = 2;
    repeated string UserTags = 3;
}

message DSearchResult {
    repeated string task_id = 1;
    string status = 2;
}